src/elm/Session/LoggedIn.elm

```diff
@@ -144,6 +155,11 @@ type alias Model =
     , hasShop : FeatureStatus
     , hasObjectives : FeatureStatus
     , hasKyc : FeatureStatus
+    , searchModel : Search.Model
+    , claimingAction : Action.Model
+
+    -- TODO: `date` is used to filter outdated actions and could be removed if we filter them on the backend
```

Comment:
i think it's safe to remove, we won't deploy this until we have Auth done, which will include that change